fastscanner/adapters/realtime/redis_channel.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from typing import Any, cast
+
+import pandas as pd
+import redis.asyncio as aioredis
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class RedisChannel:
+    def __init__(
+        self,
+        host="localhost",
```

Comment:
Let's type everything and don't use defaults.
host: str,
port: int,
The caller needs to pass these variables from the config file.